### PR TITLE
Support for diameter-dictionary module from new diameter 0.9.3 version

### PR DESCRIPTION
**Pull request by Thommil**
The new version of diameter is using a separate package for dictionary, I've just updated the dependencies to make it work

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "diameter-avp-object",
-  "version": "1.0.6",
```
Comment (atesgoral) [inline package.json]: Could you please revert this line? We bump up versions using the `npm version` command after changes are merged in. Besides, we may want to bump up the minor version for this change.

**Comment by atesgoral** [comment]
Thanks for this patch!

File: package.json (hunk 1 above)
Comment (Thommil) [reply package.json]: Done, sorry, my bad.

**Comment by atesgoral** [comment]
v1.0.7 published

**Pull request merged**